src/java.base/share/classes/java/lang/foreign/MemoryAddress.java

```diff
@@ -124,6 +124,11 @@
      * sequences with this charset's default replacement string.  The {@link
      * java.nio.charset.CharsetDecoder} class should be used when more control
      * over the decoding process is required.
+     * <p>
```

Comment:
This is unrelated, but I realized that this method javadoc was missing the "restricted" section.